src/docs/product/sentry-basics/key-terms.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Key Terms
+sidebar_order: 05
+description: "Learn about the key concepts and features of Sentry."
+---
+
+The key terms and concepts defined on this page are the fundamentals of Sentry. While we define new terms as you come across them throughout our documentation, having an understanding of the key concepts and features here will make it easier to learn more complex ideas about the product.
+
+This list is ordered so that each definition builds on the previous ones.
+
+## Key Concepts
+
+Some of the following key concepts have corresponding features with the same name listed (in upper case) in the [Key Features](#key-features) section.
+
+- **Sentry SDK** - Sentry's software development kit. SDKs are programming language/framework-specific. When you instrument your application with one of our SDKs, _event_ data from your application is captured, or sent to Sentry, so we can provide you with error and transaction information about your software.
```

Comment:
```suggestion
- **Sentry SDK** - Sentry's software development kit. SDKs are programming language/framework-specific. When you instrument your application with one of our SDKs, _event_ data from your application is captured and sent to Sentry, so we can provide you with error and performance reports.
```

Just some small wordsmithing suggestions. I'll admit that I also struggle a bit with using "software development kit" as the fundamental definition. OTOH, yes, obviously, that's literally what the acronym stands for. OTOH, it's only a helpful definition if "software development kit" holds more meaning for people than "SDK," and my guess is that it's actually the other way around (IOW, my sense is that of the two, "SDK" is the more widely known term). So you might consider something like this:

```suggestion
- **Sentry SDKs** - Sentry's programming language/framework-specific libraries for application monitoring. When you instrument your application with one of our SDKs, _event_ data from your application is captured and sent to Sentry, so we can provide you with error and performance reports.
```

Finally, I wonder if there's anyone who doesn't know what an SDK is but does know what it means to "instrument" code. (I'd guess not.) So maybe:

```suggestion
- **Sentry SDKs** - Sentry's programming language/framework-specific libraries for application monitoring. When you add one of our SDKs to your application, _event_ data from your application is captured and sent to Sentry, so we can provide you with error and performance reports.
```

Reply:
It hurts my tech writer soul to not define an abbreviation, but your point makes sense. And for some reason I thought we used the term "instrument" in this context in the docs, but having searched, I don't see it anywhere, so this works for me.